invenio_vocabularies/contrib/funders/mappings/os-v1/funders/funder-v2.0.0.json

```diff
@@ -14,8 +14,8 @@
           "type": "custom",
           "char_filter": ["strip_special_chars"],
           "filter": [
-            "lowercase",
-            "asciifolding",
+            "lowercasepreserveoriginal",
```

Comment:
@wgresshoff could I ask you to move all the changes to funder-v3.0.0.json files?
in case of breaking changes like these we need to upgrade the version of the mapping, otherwise instance upgrades are complex

Reply:
Since in practice this doesn't break existing indices created with the old mapping but only affects search behavior/accuracy, I feel we can merge without bumping the mapping version.

This means that for deployment, one needs to either:

- Do the usual recreation of indices (with downtime)
- Do a [live migration](https://github.com/inveniosoftware/docs-invenio-rdm/issues/696) with some special handling since the old and new index name will be the same (only the timestamp suffix changes).

Reply:
We used the recreation of indices with downtime and the missing funders were reindexed without problem.